device_doctor/lib/src/device.dart

```diff
@@ -0,0 +1,98 @@
+// Copyright 2020 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
```

Comment:
Docs? we may want to document this module as the one holding the interfaces/abstract classes definition.